src/components/DataGroups/TextDataGroup.js

```diff
@@ -1,21 +1,27 @@
+import { Tooltip } from "../Common/Tooltip/Tooltip";
+
 const valueToString = (value) => {
   if (typeof value === "string") return value;
 
   try {
     return JSON.stringify(value);
   } catch {
-    return value.toString();
+    return (value ?? "").toString();
```

Comment:
what kind of value could be here? it won't fail on null or undefined and that's the only values triggering nullish coalescing.

Reply:
it could be any non-JSON value - in this case null and undefined should render as "", while 0 and false should render as the "0" and "false", every thing else should render as what ever it is, be it string, int, etc